dashboard/app/models/game.rb

```diff
@@ -301,7 +296,6 @@ def channel_backed?
     CurriculumReference:curriculum_reference
     Map:map
     CustomFlappy:flappy
-    Scratch:scratch
```

Comment:
The order of this list matters, so removing this entry will probably break dance, spritelab, bubblechoice, and fish.

Reply:
Whats the best way to deal with this then? Then leave the entry?

Reply:
Yeah probably just leave it and add a comment